zebra/zebra_vty.c

```diff
@@ -3169,11 +3169,11 @@ DEFUN (no_vrf_vni_mapping,
 }
 
 /* show vrf */
-DEFUN (show_vrf_vni,
+DEFPY (show_vrf_vni,
        show_vrf_vni_cmd,
-       "show vrf vni [json]",
+       "show vrf <NAME$vrf_name|all$vrf_all> vni [json]",
```

Comment:
Isn't this like a breaking change? `show vrf vni` versus `show vrf <NAME> vni`.

Reply:
Though majority of the places when user does not provide vrf name its assumed to be `default`. Earlier was displaying all vrf information when `show vrf vni` is entered. 
I have made optional and restored previous behavior.